### DISCUSSION TITLE
第4节课作业(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#241] [作业]
Here

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135553448

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec4/

**Comment by Jeff-Huang-SHU** [comment]
https://blog.csdn.net/2301_77209094/article/details/135576229?spm=1001.2014.3001.5502

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/1a1dc7130573423eaccd3c58c566e2b0

**Comment by Smiling-Weeping-zhr** [comment]
https://www.cnblogs.com/smiling-weeping-zhr/p/17961091

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135585013?spm=1001.2014.3001.5502

**Comment by canxingliuying** [comment]
https://blog.csdn.net/weixin_42449385/article/details/135587243?spm=1001.2014.3001.5501

**Comment by ltstar** [comment]
[课程4-xtuner.md](https://github.com/InternLM/tutorial/files/13931535/4-xtuner.md)

**Comment by siuserjy** [comment]
https://blog.csdn.net/weixin_46334272/article/details/135661136

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/FOVpdYLICoMAZZx3h1fczSprnyc

**Comment by JiangXiaobai00** [comment]
https://d0elb2210mg.feishu.cn/docx/At2QdTRxdoBOVZxsD87cjQJVnbg?from=from_copylink

**Comment by sunhanyu714** [comment]
https://github.com/sunhanyu714/Internlm_learning/blob/main/%E7%AC%AC%E5%9B%9B%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by 0-yy-0** [comment]
https://blog.csdn.net/qq_34354951/article/details/135834112
